commands/scale/cluster/command.go

```diff
@@ -107,13 +107,13 @@ type Result struct {
 
 // getConfirmation asks the user for confirmation for scaling actions.
 func getConfirmation(args Arguments, maxBefore int, minBefore int, currentWorkers int) error {
-	if int64(currentWorkers) > args.WorkersMax && args.WorkersMax == args.WorkersMin {
+	if int64(currentWorkers) > args.WorkersMax && args.WorkersMax == *args.WorkersMin {
```

Comment:
what happens in case WorkersMin is nil? Do we need to check it here again or is the error handled before?